src/routes/applicationsContentRoute.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const router = express.Router();
+const applicationsController = require('../controllers/applicationsContentController');
+const registerPartials = require('../middlewares/registerPartials');
+
+router.get('/((?!favicon.ico)):orgName/applications', registerPartials, applicationsController.loadApplications);
+router.get('/((?!favicon.ico)):orgName/applications/create', registerPartials, applicationsController.loadThrottlingPolicies);
+router.get('/((?!favicon.ico)):orgName/applications/:applicationid', registerPartials, applicationsController.loadApplication);
+router.get('/((?!favicon.ico)):orgName/applications/:applicationid/edit', registerPartials, applicationsController.loadApplicationForEdit);
+
+router.post('/applications', applicationsController.saveApplication);
```

Comment:
what is the purpose of these routes (without the orgName)

Reply:
These are not get methods. These routes are being used to get methods to backend from frontend.